src/vue/modules/coinpayments-deposit/components/pending-issuances-table.vue

```diff
@@ -21,6 +24,25 @@ export default {
   },
   props: {
     pendingIssuances: { type: Array, required: true },
+    issuanceCreated: { type: Boolean, required: true },
+  },
+  data: () => ({
+    selectedIssuance: null,
+  }),
+  watch: {
+    issuanceCreated (value) {
+      if (value) {
+        this.resetIssuanceSelection()
+      }
+    },
```

Comment:
instead of weak `issuanceCreated` prop I’d recommend watching of `pendingIssuances` length

Reply:
But It won't help, if Alice wants to create/generate another QR, I need to know wether she submitted new request or not in coinpayments-form, in order to close other expanded QR codes

For example pendingIssuances is already has length of 2 and Alice creates new address, pendingIssuances would still have length of 2 until Alice reset page and newly created QR code goes to pendingIssuances thus increasing its length to 3